cps/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition.java

```diff
@@ -128,32 +123,8 @@ public FormValidation doCheckScript(@QueryParameter String value, @QueryParamete
             return sandbox ? FormValidation.ok() : ScriptApproval.get().checking(value, GroovyLanguage.get());
         }
 
-        @Restricted(DoNotUse.class) // j:invokeStatic does not consistently work on plugin classes
-        public Collection<? extends StepDescriptor> getStepDescriptors() {
-            return StepDescriptor.all();
-        }
-
-        @Restricted(DoNotUse.class) // from config.jelly
-        public HttpResponse doGenerateSnippet(StaplerRequest req, @QueryParameter String json) throws Exception {
-            // TODO is there not an easier way to do this?
-            JSONObject jsonO = JSONObject.fromObject(json);
-            Jenkins j = Jenkins.getInstance();
-            if (j == null) {
-                throw new IllegalStateException("Jenkins is not running");
-            }
-            Class<?> c = j.getPluginManager().uberClassLoader.loadClass(jsonO.getString("stapler-class"));
-            Object o;
-            try {
-                o = req.bindJSON(c, jsonO);
-            } catch (RuntimeException x) { // e.g. IllegalArgumentException
-                return HttpResponses.plainText(Functions.printThrowable(x));
-            }
-            try {
-                return HttpResponses.plainText(Snippetizer.object2Groovy(o));
-            } catch (UnsupportedOperationException x) {
-                Logger.getLogger(CpsFlowExecution.class.getName()).log(Level.WARNING, "failed to render " + json, x);
-                return HttpResponses.plainText(x.getMessage());
-            }
+        public Snippetizer getSnippetizer() {
+            return Snippetizer.get();
```

Comment:
You can just inject Snippetizer as a field like `@Inject private Snippetizer snippetizer;` and avoid static `get` method.